server/src/main/java/org/opensearch/search/aggregations/metrics/SumAggregationBuilder.java

```diff
@@ -101,8 +110,22 @@ protected ValuesSourceType defaultValueSourceType() {
     }
 
     @Override
-    protected void innerWriteTo(StreamOutput out) {
-        // Do nothing, no extra state to write to stream
+    protected void innerWriteTo(StreamOutput out) throws IOException {
+        boolean hasMethod = method != null;
+        out.writeBoolean(hasMethod);
```

Comment:
```suggestion
        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
           out.writeOptionalString(method);
         }
```

Reply:
updated in https://github.com/opensearch-project/OpenSearch/pull/6984/commits/7d50f500d045f1c8acb3d71318ef777cef6e6143